apps/src/applab/designMode.js

```diff
@@ -151,6 +152,18 @@ designMode.onPropertyChange = function(element, name, value) {
   designMode.editElementProperties(element);
 };
 
+function renderIcon(value) {
```

Comment:
This could use a function level comment. I also wonder if this is the right name for this. If I understand it correctly, calling this method gives me information about how to render the icon, but the actually rendering doesn't occur until we do something like background-image: url(<result>).